server/chat-plugins/quotes.ts

```diff
@@ -139,7 +139,7 @@ export const pages: PageTable = {
 			return `${buffer}<h2>This room has no quotes.</h2></div>`;
 		}
 
-		buffer += `<h2>Quotes for ${room.title} (${roomQuotes.length}):</h2>`;
+		buffer += `<h2>Quotes for ${Utils.escapeHTML(room.title)} (${roomQuotes.length}):</h2>`;
```

Comment:
```suggestion
		buffer += Utils.html`<h2>Quotes for ${room.title} (${roomQuotes.length}):</h2>`;
```

Reply:
dont see any difference but whatever

Reply:
performance wise mine might be a little tiny bit faster since the roomQuotes.length part can never be anything but ``/[0-9]+/``